bps/templates/bps_biasBootstrap.yaml

```diff
@@ -0,0 +1,15 @@
+pipelineYaml: "${CP_PIPE_DIR}/pipelines/${INSTRUMENT}/cpBiasBootstrap.yaml"
+
+project: "${TICKET}"
+campaign: "${TICKET}"
+
+biasBootstrapCollection: "${USER_CALIB_PREFIX}$INSTRUMENT/calib/$TICKET/$TAG/biasBootstrapGen.$RERUN"
+biasBootstrapRun: "{biasBootstrapCollection}/run${BOOTSTRAP_RUN_NUMBER}"
```

Comment:
Is this missing a `$` for `biasBootstrapCollection`?

Reply:
This is a bps variable, and must be quoted because yaml gets confused otherwise.